src/store/ChartStore.js

```diff
@@ -224,6 +224,12 @@ class ChartStore {
         this.holderStyle = stxx.chart.panel.holder.style;
 
         stxx.append('deleteHighlighted', this.updateComparisons);
+
+        stxx.append('adjustPanelPositions', function () {
+            const panel = Object.keys(stxx.panels)[1];
+            stxx.panels[panel].up.style.display = 'none';
```

Comment:
@MahboobehMohammadi LGTM. just one simple modification. check to see if `stxx.panels[panel]`is defined or not. maybe the user does not select any indicator.